lang/en.js

```diff
@@ -136,6 +127,33 @@ var LANGUAGE = {
     },
     'zh-TW': {
       'title': 'Chinese (traditional, Taiwan)'
+    },
+    'hr': {
+      'title': 'Croatian'
+    },
+    'pl': {
+      'title': 'Polish'
+    },
+    'es': {
+      'title': 'Spanish'
+    },
+    'fr': {
+      'title': 'French'
+    },
+    'ko': {
+      'title': 'Korean'
+    },
+    'nb': {
+      'title': 'Norwegian Bokmal'
+    },
+    'nn': {
+      'title': 'Norwegian Nynorsk'
+    },
+    'pt': {
+      'title': 'Portugese'
```

Comment:
I don't think this is right, is this a misspelling of the English word? Or is it meant to be the Portuguese translation (in that case I don't think it's right, but I'm not 100% sure).

Reply:
Thanks Carlos. It was a spelling mistake. Fixed

Reply:
Thanks Mark. Look like the typo is also in the editor.html file, could you fix that one as well? https://github.com/bbcmicrobit/PythonEditor/blob/b2876341d0d91a4887caf65f0054f8857c2a7d08/editor.html#L357